scripts/apps/users/views/user-preferences.html

```diff
@@ -121,7 +121,7 @@ <h3 id="article-defaults" class="sd-heading sd-text-align--left sd-text--sans sd
                 </li>
 
                 <li class="simple-list__item simple-list__item--stacked simple-list__item--justify-flex-start" ng-hide="profileConfig.category === false">
-                    <h3 id="prefered-categories" class="sd-heading sd-text-align--left sd-text--sans sd-heading--h3 sd-txt-transform--capitalize">{{ preferences['categories:preferred'].category }}</h3>
+                    <h3 id="prefered-categories" class="sd-heading sd-text-align--left sd-text--sans sd-heading--h3 sd-txt-transform--capitalize" translate>{{ preferences['categories:preferred'].category }}</h3>
```

Comment:
here we shouldn't use it

Reply:
yep, @devketanpro translate attributes can only be used on inline strings. If it's a variable, we can't add a `translate` attribute and have to ensure that string is translated before it goes to a variable.